frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -79,52 +88,83 @@ import { operatorGroupFor, operatorNamespaceFor } from './operator-group';
 import { SubscriptionDetails, catalogSourceForSubscription } from './subscription';
 import { ClusterServiceVersionLogo, referenceForProvidedAPI, providedAPIsFor } from './index';
 
+const clusterServiceVersionStateToProps = (state: RootState): ClusterServiceVersionStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
+  };
+};
+
 const isSubscription = (obj) => referenceFor(obj) === referenceForModel(SubscriptionModel);
 const isCSV = (obj) => referenceFor(obj) === referenceForModel(ClusterServiceVersionModel);
 const tableColumnClasses = [
   '',
   '',
   classNames('pf-m-hidden', 'pf-m-visible-on-sm'),
   classNames('pf-m-hidden', 'pf-m-visible-on-lg'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-2xl'),
   classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
   Kebab.columnClass,
 ];
 
-export const ClusterServiceVersionTableHeader = () => {
-  return [
-    {
-      title: 'Name',
-      sortField: 'metadata.name',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[0] },
-    },
-    {
-      title: 'Namespace',
-      sortField: 'metadata.namespace',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[1] },
-    },
-    {
-      title: 'Status',
-      props: { className: tableColumnClasses[2] },
-    },
-    {
-      title: 'Deployment',
-      sortField: 'spec.install.spec.deployments[0].name',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[3] },
-    },
-    {
-      title: 'Provided APIs',
-      props: { className: tableColumnClasses[4] },
-    },
-    {
-      title: '',
-      props: { className: tableColumnClasses[5] },
-    },
-  ];
+const nameHeader: Header = {
+  title: 'Name',
+  sortField: 'metadata.name',
+  transforms: [sortable],
+  props: { className: '' },
```

Comment:
You should use the values from `tableColumnClasses` or remove it. Maybe it would be better to have constants for each (since the index will be different depending on whether all namespaces is selected).

```ts
const nameColumnClass = '';
```

Then...

```suggestion
  props: { className: nameColumnClass },
```

etc.